src/pages/settings-resources/forms/ReportLayoutsForm.js

```diff
@@ -0,0 +1,198 @@
+
+    const response2Map = new Map(response2List.map(item => [item.id, item.isInactive]))
+
+    const updatedList = responseList.map(item => {
+      return {
+        ...item,
+        isInactive: response2Map.has(item.id) ? response2Map.get(item.id) : false
+      }
+    })
+
+    return {
+      count: updatedList.length,
+      list: updatedList,
+      statusId: 1,
+      message: ''
+    }
+  }
+
+  const fetchData = async () => {
+    const response = await getRequest({
+      extension: SystemRepository.ReportLayout,
+      parameters: `_resourceId=${row.resourceId}`
+    })
+
+    const response2 = await getRequest({
+      extension: SystemRepository.ReportLayoutObject.qry,
+      parameters: `_resourceId=${row.resourceId}`
+    })
+
+    const processedData = processResponses(response, response2)
+    setData(processedData || [])
+  }
+
+  const { formik } = useForm({
+    enableReinitialize: true,
+    validateOnChange: true,
+
+    initialValues: {
+      resourceId: row.resourceId,
+      resourceName: row.resourceName
+    },
+    onSubmit: async obj => {
+      const inactiveItems = data.list
+        .filter(item => item.isInactive)
+        .map(item => ({
+          resourceId: row.resourceId,
+          id: item.id,
+          isInactive: item.isInactive
+        }))
+
+      const payload = {
+        resourceId: row.resourceId,
+        items: inactiveItems
+      }
+
+      await postRequest({
+        extension: SystemRepository.ReportLayoutObject.set2,
+        record: JSON.stringify(payload)
+      })
+
+      toast.success(platformLabels.Updated)
+
+      w.close()
+    }
+  })
+
+  const returnURL = async rowData => {
+    const response = await getRequest({
+      extension: 'KVS.asmx/getAttachment',
+      parameters: `_resourceId=${formik.values.resourceId}&_layoutId=${rowData.id}`
+    })
+
+    return response.record.url
+  }
+
+  useEffect(() => {
+    fetchData()
+  }, [])
+
+  const columns = [
+    {
+      field: 'api',
+      headerName: labels.api,
+      flex: 1
+    },
+    {
+      field: 'instanceName',
+      headerName: labels.instanceName,
+      flex: 1
+    },
+
+    {
+      field: 'parameters',
+      headerName: labels.params,
+      flex: 1
+    },
+    {
+      field: 'layoutName',
+      headerName: labels.layoutName,
+      flex: 1
+    },
+    {
+      width: 100,
+
+      cellRenderer: row => {
+        return (
+          <Box sx={{ display: 'flex', width: '100%', justifyContent: 'center' }}>
+            <Typography
+              variant='body2'
+              component='a'
+              onClick={async () => {
+                const url = await returnURL(row.data)
+                window.open(url, '_blank')
+              }}
+              target='_blank'
+              rel='noopener noreferrer'
+              sx={{
+                color: 'blue',
+                textDecoration: 'underline',
+                cursor: 'pointer',
+                fontSize: '17px'
```

Comment:
make the font smaller for whn we merge the fields task the word could still be visible